osu.Game/Overlays/TabControlOverlayHeader.cs

```diff
@@ -106,7 +106,19 @@ private class OverlayHeaderTabItem : OverlayTabItem
                 public OverlayHeaderTabItem(T value)
                     : base(value)
                 {
-                    Text.Text = ((Value as Enum)?.GetDescription() ?? Value.ToString()).ToLower();
+                    if (!(Value is Enum enumValue))
+                        Text.Text = Value.ToString().ToLower();
+                    else
+                    {
+                        var localisableDescription = enumValue.GetLocalisableDescription();
+                        var nonLocalisableDescription = enumValue.GetDescription();
+
+                        // If localisable == non-localisable, then we must have a basic string, so .ToLower() is used.
+                        Text.Text = localisableDescription.Equals(nonLocalisableDescription)
+                            ? nonLocalisableDescription.ToLower()
+                            : localisableDescription;
+                    }
```

Comment:
Is this one actually required, or was that a shotgun change across all `.GetDescription()` usages? Because it doesn't seem like the former.

Eventually we'll probably need to support localisable text transformations like this somehow, because web relies on them via CSS I'm pretty sure...

Reply:
It was a shotgun change over the ones I thought made sense for the time being, especially around "overlay" components.

Reply:
Hm, ok. I'm fine with leaving, may as well address the transformation part when it becomes an issue. Was wondering about it more than anything.